src/client/features/interactions/index.js

```diff
@@ -66,12 +73,17 @@ class Interactions extends React.Component {
         loaded:_.assign(this.state.loaded,{network:true})
       });
     });
-    //get ids from uris
-    const geneIds = sources.map(source=>
-      source.includes('pathwaycommons')?
-      ServerAPI.pcQuery('traverse',{uri:source,path:`${_.last(source.split('/')).split('_')[0]}/displayName`}).then(result=>result.json())
-      .then(id=> _.words(id.traverseEntry[0].value[0]).length===1 ? id.traverseEntry[0].value[0].split('_')[0] : ''):
-      source.replace(/\//g,' ')
+    //get ids from uris 
+    const geneIds = sources.map(source =>
```

Comment:
What case was this traverse trying to handle?

Reply:
It says "get ids from uris" in the comment line above that code (if, e.g., an uri is standard PR's uri, I bet); @logmosier can comment on exact purpose of this. I did not change the purpose of this code. It would be great to have more comments in the code, in many places.